app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_parser.rb

```diff
@@ -0,0 +1,26 @@
+module ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventParser
+  def self.event_to_hash(event, ems_id)
+    event_hash = {
+      :event_type => event["action"],
+      :source     => "IBM_CLOUD_VPC",
+      :ems_ref    => ems_id,
```

Comment:
`:ems_ref` should be the unique identifier of _this event_

Does the `event` have some sort of unique id?

Reply:
Yes there seems to be a unique identifier:
```
> event["_id"]
"1376197173894295555"
```

Is there any use for `@cfg[:ems_id]` in this method?

Reply:
> Is there any use for @cfg[:ems_id] in this method?

Yes the event table has an `:ems_id` column so it would be good to set that